### PR TITLE
Event manager

### PR DESCRIPTION
**Pull request by r-mllr**
added ability to add users to events

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -1,3 +1,14 @@
 class Event < ApplicationRecord
+  has_and_belongs_to_many :users
```
Comment (fate83) [inline app/models/event.rb:2]: You probably need another association here to track who created the event.

File: spec/requests/events_spec.rb [hunk 2]
```diff
@@ -0,0 +1,184 @@
+require 'rails_helper'
+require 'request_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
```
Comment (fate83) [inline spec/requests/events_spec.rb:4]: Comment can be deletef

**Review by fate83 — APPROVED** [review]
Left 2 comments but works without them

File: app/models/event.rb (hunk 1 above)
Comment (r-mllr) [reply app/models/event.rb:2]: Just for tracking or also for other purposes?

**Pull request merged**